Library/Homebrew/test/livecheck_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "livecheck"
+
+describe Livecheck do
+  subject(:livecheckable) { described_class.new }
+
+  describe "#regex" do
+    it "returns nil if unset" do
+      expect(livecheckable.regex).to be nil
+    end
+
+    it "returns the Regex if set" do
+      livecheckable.regex(/foo/)
+      expect(livecheckable.regex).to eq(/foo/)
+    end
+  end
+
+  describe "#skip" do
+    it "sets the instance variable skip to true and skip_msg to nil when no arguments are given" do
+      livecheckable.skip
+      expect(livecheckable.instance_variable_get(:@skip)).to be true
+      expect(livecheckable.instance_variable_get(:@skip_msg)).to be nil
+    end
+
+    it "sets the instance variable skip to true and skip_msg to nil when the argument is blank" do
```

Comment:
Can skip this test as it's essentially just testing the behaviour of `blank?` (and isn't needed if you use `.presence`)